dashboard/app/models/ability.rb

```diff
@@ -65,7 +65,7 @@ def initialize(user)
         can :manage, UserLevel do |user_level|
           !user.students.where(id: user_level.user_id).empty?
         end
-        can :read, Plc::UserCourseEnrollment
+        can :index, Plc::UserCourseEnrollment
```

Comment:
Why change it to index? That's just an alias for read. See https://github.com/CanCanCommunity/cancancan/wiki/Action-Aliases

Unless you're specifically checking for that custom action (though I'm not sure how that will work with the alias).